target/linux/ramips/image/Makefile

```diff
@@ -11,20 +11,33 @@ DEVICE_VARS += BUFFALO_TAG_PLATFORM BUFFALO_TAG_VERSION BUFFALO_TAG_MINOR
 DEVICE_VARS += SEAMA_SIGNATURE SEAMA_MTDBLOCK
 DEVICE_VARS += SERCOMM_HWNAME SERCOMM_HWID SERCOMM_HWVER SERCOMM_SWVER
 DEVICE_VARS += SERCOMM_PAD JCG_MAXSIZE
+DEVICE_VARS += SERCOMM_KERNEL_OFFSET SERCOMM_ROOTFS_OFFSET
 
 loadaddr-y := 0x80000000
 loadaddr-$(CONFIG_TARGET_ramips_rt288x) := 0x88000000
 loadaddr-$(CONFIG_TARGET_ramips_mt7621) := 0x80001000
+loadaddr-$(CONFIG_TARGET_ramips_mt7621_DEVICE_beeline_smartbox-giga) := \
+	0x81001000
```

Comment:
This does not work. This is only evaluated once per (sub)target, so if you built multiple devices, all will have the same value ...
Same for the change below.

Reply:
Ok. Fixed.